### PR TITLE
Feature article

### PR DESCRIPTION
**Pull request by erazera**
Added the article feature, then improved it by adding the metrics.

### CODE REVIEW

File: assets/css/article.css
Comment (guilhermehoff) [inline assets/css/article.css:1]: good start of work

File: assets/css/styles.css
Comment (guilhermehoff) [inline assets/css/styles.css:1]: good desktop version
the mobile version must be made!!

File: assets/css/article.css
Comment (erazera) [reply assets/css/article.css:1]: thanks

File: assets/css/styles.css
Comment (erazera) [reply assets/css/styles.css:1]: ok, will work on that

**Review by guilhermehoff — APPROVED** [review]
PR aproved

**Pull request merged**